integreat_cms/core/signals/feedback_signals.py

```diff
@@ -61,7 +61,12 @@ def feedback_create_handler(sender: ModelBase, **kwargs: Any) -> None:
     :param sender: The class of the feedback that was deleted
     :param \**kwargs: The supplied keyword arguments
     """
-    if (instance := kwargs.get("instance")) and (
-        feedback_ptr := getattr(instance, "feedback_ptr", None)
-    ):
-        invalidate_obj(feedback_ptr)
+    try:
+        if (instance := kwargs.get("instance")) and (
+            feedback_ptr := getattr(instance, "feedback_ptr", None)
+        ):
+            invalidate_obj(feedback_ptr)
+    except Feedback.DoesNotExist:
+        # When installing fixtures, the related feedback might not yet be initialized.
```

Comment:
I assume that the occurrence of this bug depends on the order in which the test data are loaded. Since the test data file got completely reshuffled with this pr, that might be a likely explanation. I do not know for sure though.

Reply:
a bit puzzling to me, but okay :)

Reply:
ah. wait, I've got an idea